cookbook/integrations/flytekit_plugins/dolt/foo/.dolt/noms/manifest

```diff
@@ -0,0 +1 @@
+4:__LD_1__:ejqr2p7bmc66dk6srorq52s8gs90kb30:o1cb6u1mfpcbploi9211sl5908nc91oj:dnpd1n3msi5v2pfcd8fn5mhcv2f1b3jd:1:62ev6psp21qe87otfpsiil67v0lc87aq:2:4bujnjuatdli0klda7ucot0tl836jujt:1:03ivi2ph93n3bacr4javbd0mephosra4:4
```

Comment:
do you want to check this in? .gitignore?

Reply:
I can delete, it's a simple setup step for people running tutorial

Reply:
no you dont have to, it looked like a binary so was not sure